dev/doc/release-process.md

```diff
@@ -182,7 +172,17 @@ Ping `@Zimmi48` to:
 
   *TODO:* automate this with coqbot.
 
-## At the patch-level release time ##
+## This is now delegated to the platform maintainers ##
 
-We generally do not update the magic numbers at this point (see
-[`2881a18`](https://github.com/coq/coq/commit/2881a18)).
+- [ ] Sign the Windows and MacOS packages and upload them on GitHub.
+  + The Windows packages must be signed by the Inria IT security
+    service. They should be sent as a link to the binary (via
+    [filesender](https://filesender.renater.fr) for example) together
+    with its SHA256 hash in a signed e-mail to `dsi.securite` @
+    `inria.fr` putting `@maximedenes` in carbon copy.
+  + The MacOS packages should be signed with our own certificate. A
+    detailed step-by-step guide can be found [on the
+    wiki](https://github.com/coq/coq/wiki/SigningReleases).
```

Comment:
```suggestion
    wiki](https://github.com/coq/coq/wiki/SigningReleases).
  + The snap package has to be built and uploaded to the snap store by running a [https://github.com/coq/platform/tree/v8.13/linux/snap/github_actions](platform CI job manually). Then ask `@gares` to publish the upload or give you the password for the `coq-team` account on the store so that you can do it yourself.
```